js/page-specific/baltcogo-app/constants/constants.js

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
exclude local sources from project, ensure local and contants match all properties.